app/src/ui-blokada/kotlin/core/Tunnel.kt

```diff
@@ -298,7 +298,7 @@ fun newTunnelModule(ctx: Context): Module {
                 }
             }
 
-            ktx.on(TunnelEvents.REQUEST) { request ->
+            ktx.on(TunnelEvents.REQUEST_SAVED) { request ->
```

Comment:
The counter for the number of blocked requests will break with this change

Reply:
Fixed.